FAQS.md

```diff
@@ -0,0 +1,54 @@
+**Markor - Frequently Asked Questions**
+
+**Markor - FAQ**
+
+This Frequently Asked Question will be continually updated to accommodate new information with respect to new updates and changes on the App and will be used inside the App to help users find answers to their questions.
+
+
+**What is Markor?**
+
+Markor is an Android application developed by Gregor Santner. It allows you to write, save and open files in markdown, plain and .txt formats. A note app developed for easy and improved note taking in markdown and .txt. 
```

Comment:
it's plaintext files, whatever file-ending it is. Markdown and toto.txt are two (and the currently only) special formats having syntax highlighting and a whole suite of text modification actions.